src/compositemark.ts

```diff
@@ -0,0 +1,64 @@
+import {Encoding} from './encoding';
```

Comment:
No -- it's not single instance.  This is a preparation for a recurring pattern.

This way we can add experimental plugins for box plot / error bar / confidence internal that we need for CompassQL and Voyager without adding them to the main API until we are happy with the syntax enough.

(In fact, in a future PR, I am leaning toward moving ERRORBAR out of here.)